extensions/src/main/java/org/pentaho/platform/web/http/filters/PentahoWebContextFilter.java

```diff
@@ -197,16 +197,10 @@ public void doFilter( ServletRequest request, ServletResponse response, FilterCh
     }
   }
 
-  private boolean shouldUseFullyQualifiedUrl( HttpServletRequest httpRequest ) {
-    final String useFullyQualifiedUrlParameter = httpRequest.getParameter( "fullyQualifiedUrl" );
-    if ( useFullyQualifiedUrlParameter != null ) {
-      return "true".equals( useFullyQualifiedUrlParameter );
-    } else {
-      final String referer = httpRequest.getHeader( "referer" );
-      final String serverAddress = httpRequest.getScheme() + "://" + httpRequest.getServerName() + ":" + httpRequest.getServerPort();
 
-      return referer != null && !referer.startsWith( serverAddress );
-    }
+  private boolean shouldUseFullyQualifiedUrl( HttpServletRequest httpRequest ) {
+    // Will return false for now.
```

Comment:
If not removing it completely, I would keep the code for the explicit use of the `fullyQualifiedUrl` parameter and just always return `false` inside the `else` clause (so it doesn't try to be _"smart"_).

Also include the BACKLOG-16728 reference in the comment, so in the future is it easier to find out why it is returning false for now.